docs/releases/status_update.md

```diff
@@ -0,0 +1,125 @@
+# ❤️Status Update ❤️ 
+
+
+## 🆕 odin status
+
+- Can be used inside the Valheim docker container.
+- Can be used to query other servers with the `--address "IP_ADDRESS"` argument.
+- Will try to check for your public facing IP, to avoid this set the `ADDRESS` environment variable to be the query address. Ex: `127.0.0.1:2457`
+- Has the ability to output with JSON, see below.
+
+### Json flag
+Without `--json`
+```shell
+[ODIN][INFO]  - Name: Creative Update
+[ODIN][INFO]  - Players: 0/10
+[ODIN][INFO]  - Map: Creative Update
+[ODIN][INFO]  - BepInEx Enabled: true
+[ODIN][INFO]  - BepInEx Mods: ValheimPlus.dll
```

Comment:
Ill leave it as is for now :P of beans request an update for that, it can be changed but for now, if they want to do `odin notify "$(odin status)"` I expect em to clean it up by using `--json` and `jq` to make the output look nice.